nipype/interfaces/fsl/model.py

```diff
@@ -1886,7 +1886,7 @@ class SmoothEstimateInputSpec(FSLCommandInputSpec):
 class SmoothEstimateOutputSpec(TraitedSpec):
     dlh = traits.Float(desc="smoothness estimate sqrt(det(Lambda))")
     volume = traits.Int(desc="number of voxels in mask")
-    resels = traits.Float(desc="number of resels")
+    resels = traits.Float(desc="number of voxels per resel")
```

Comment:
I've had difficulty tracking down documentation from FSL on this, but found the following footnote in a @nicholst [white paper](https://www.fmrib.ox.ac.uk/datasets/techrep/tr08tn1/tr08tn1.pdf):

> Beware that FSL's `smoothest` program reports `RESELS`, which is actually the the size of one Resel, i.e. volume of space with dimensions FWHM_x, FWHM_y, FWHM_z, i.e. from `smoothest`, `RESELS` = FWHMx FWHMy FWHMz. In SPM, “RESEL” generically refers to the Resel count, R_3. And, for completeness, note that `smoothest`'s DHL=|Λ|^1/2, “Determinant Half-Power Lambda.”

Further, if I'm reading it correctly, it seems that these FWHM are in voxels for each dimension, so I think "number of voxels per resel" is a correct phrasing, but it might be clearer to write:

```suggestion
    resels = traits.Float(desc="volume of resel, in voxels, defined as FWHM_x * FWHM_y * FWHM_z")
```